src/coveralls.jl

```diff
@@ -156,14 +156,15 @@ module Coveralls
     end
 
     """
-        submit_token(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
+        submit_local(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
 
     Take a `Vector` of file coverage results (produced by `process_folder`),
     and submits them to Coveralls. For submissions not from TravisCI.
 
     git_info can be either a `Dict` or a function that returns a `Dict`.
     """
-    function submit_token(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
+
```

Comment:
This line break should be removed, otherwise the docstring doesn't apply to the function definition below it.

Reply:
I think that was due to my rebase.